tests/unit/Entity/EntityIdValueTest.php

```diff
@@ -75,41 +78,95 @@ public function testGetSortKey( EntityIdValue $id ) {
 		$this->assertInternalType( 'string', $id->getSortKey() );
 	}
 
+	public function provideGetArrayValue() {
+		return [
+			'Q2147483647' => [
```

Comment:
The key is shown when the test fails. Without it, there is no good way to see while case failed.